### PR TITLE
Simplification of README

### LINKED ISSUE #784: Proposal: Gitter room
I think we should just leave the GitHub issue tracker for bugs and suggestions and have a Gitter room for doubts and questions instead. That will prevent the flooding on GitHub.
Thoughts?:thought_balloon:

### PR DESCRIPTION
**Pull request by sntdevco**
This will resolve #784

### CODE REVIEW

**Comment by sntdevco** [comment]
@azat What do you think?

File: README.md [hunk 1]
```diff
@@ -209,208 +211,4 @@ use and development:
...
- * Simone Basso
- * Vlad Shcherban
- * Tim Hentenaar
- * Breaker
- * johnsonlee
- * Philip Prindeville
- * Vis Virial
-
-If we have forgotten your name, please contact us.
+fixing bugs: [See CONTRIBUTORS](./CONTRIBUTORS.MD)
```
Comment (azat) [inline README.md]: I think that it's better to use absolute path here, since it should mean the same

**Comment by azat** [comment]
I don't think that it is useful to put add/delete of the contributors into separate commits, can you squash them? (only two, not three)

Also looks like this will not resolve the problem 100%, maybe have time to do [this](https://github.com/libevent/libevent/issues/784#issuecomment-470623880) ?

(But I guess this can be a start, so I can merge it as-is)

File: README.md (hunk 1 above)
Comment (sntdevco) [reply README.md]: All right, I'll squash all commits and replace that with an absolute path 😄
I'll also start working on the Docs as you had asked

**Comment by sntdevco** [comment]
I'm closing this PR and reopening a new one (rebase is not functioning as there's some token problem, so I'll fork again and do the commits)

**Comment by azat** [comment]
Sounds great! Thanks!

**Comment by sntdevco** [comment]
@azat Should I include your name and mine? (Seems it's missing)

**Comment by azat** [comment]
Skip my (since I already there at least) and include yours off course

**Pull request closed without merging**